### PR TITLE
update features for internal use of Jakarta Connectors API

### PR DESCRIPTION
**Pull request by njr-11**
Now that the Jakarta Connectors API JAR has been brought into Liberty, update the jca.cm, jdbc, and jcaSecurity features for Jakarta to use it instead of Java EE.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jcaSecurity-1.0.feature [hunk 1]
```diff
@@ -1,7 +1,7 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
 symbolicName=com.ibm.websphere.appserver.jcaSecurity-1.0
 -features=com.ibm.websphere.appserver.containerServices-1.0
--bundles=com.ibm.ws.security.jca, \
+-bundles=\
```
Comment (njr-11) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.jcaSecurity-1.0.feature:4]: This corrects a mistake in a previous delivery.  The inclusion of this bundle vs the jakarta version of it is done by the autofeature, and should not be here in the main jcaSecurity feature.

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by njr-11** [comment]
The first build was good. However, there was a merge conflict requiring another build.
In the meantime, I've discovered more jdbc bundles with indirect dependencies on Java EE that need to be transformed as well, so I've added a second commit for that since it needs to rebuild anyway.

#build
#run-libby-bot

File: dev/com.ibm.ws.jdbc.4.2/bnd.bnd [hunk 2]
```diff
@@ -16,6 +16,8 @@ Bundle-Name: Database Connectivity
 Bundle-SymbolicName: com.ibm.ws.jdbc.4.2
 Bundle-Description: Database connectivity infrastructure, version ${bVersion}
 
+jakartaeeMe: true
```
Comment (nmittles) [inline dev/com.ibm.ws.jdbc.4.2/bnd.bnd:19]: Copyright header needs to be updated.

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
#build

**Comment by njr-11** [comment]
Build has 5 test failure which are duplicate of pre-existing issues in unrelated test buckets.

**Pull request merged**